src/App.vue

```diff
@@ -18,23 +18,33 @@ export default {
 
   data() {
     return {
-      timestamp: Date.now(),
+      timestamp: new Date().toLocaleDateString(),
       testNotes: [
       {
-          timestamp: new Date(),
+          timestamp: new Date("7/6/2023").toLocaleDateString(),
           tags: ["tag 1", "tag 2"],
           message: "This is the note body message.",
         },
-        {
-          timestamp: new Date(),
-          tags: ["tag 1", "tag 3"],
-          message: "This is the note TWO body message.",
-        },
+       // {
+         // timestamp: new Date(),
+          //tags: ["tag 1", "tag 3"],
+          //message: "This is the note TWO body message.",
+        //},
       ],
     };
   },
 
-  methods: {},
+  methods: {
+		addNewNote(noteData) {
+			this.testNotes.push ({ 
```

Comment:
@alvarix 
I think you should be able to just push the entire noteData object. You can't right now because you've used an "entry" prop where I've used "message". We can standardize to "entry" in another commit later.

Reply:
oh, bcs the prop names dont match.
specifically though, I decided to make the tags input a string, which is converted into an array when I push it into the new note.

not sure if this is the best practice but I found it easier at the time.
ditto with date.